docs/reference/specifications/in-process-providers.md

```diff
@@ -94,6 +95,15 @@ This evaluator selects a variant based on whether the specified property within
 starts/ends with a certain string.
 For more specific implementation guidelines, please refer to [this document](./custom-operations/string-comparison-operation-spec.md).
 
+### $flagd properties in the evaluation context
+
+An in-process flagd provider should also add the following properties to the JsonLogic evaluation context so that users can use them in their targeting rules.
+
+| Property | Description |
+|----------|-------------|
+| `$flagd.flagKey` | The name of the flag key |
```

Comment:
Please update this too, once we've agreed to a description in the flag definition file.

Reply:
Updated using Todd's suggestion. Thank you!